tests/integration/test_branch_integration.py

```
@@ -0,0 +1,369 @@
+            'context_chunking',
+            'tiktoken',
+            'summarization',
+            'max_tokens',
+            'chunk_size',
+        ]
+        
+        # Check these aren't in workflows anymore
+        workflow_files = list(Path(".github/workflows").glob("*.yml"))
+        
+        for wf_file in workflow_files:
+            with open(wf_file, 'r') as f:
```

Ruff (0.14.7)

Unnecessary mode argument

Remove mode argument

(UP015)